bin/bam2nucleosome

```diff
@@ -39,7 +40,11 @@ def parseArguments(args=None):
                         default = None,
                         type = int,
                         required = False)
-
+    parser.add_argument('--smoothLength',
+                        help = "length of smoothing to be done.",
```

Comment:
I think help line will have to be elaborated a bit on this arg, but I left it to fidel..